doc/modules/ensemble.rst

```diff
@@ -98,14 +98,20 @@ controls the number of iterations of the boosting process::
   >>> clf.score(X_test, y_test)
   0.8965
 
-Available losses for regression are 'squared_error',
-'absolute_error', which is less sensitive to outliers, and
-'poisson', which is well suited to model counts and frequencies. For
-classification, 'log_loss' is the only option. For binary classification it uses the
-binary log loss, also known as binomial deviance or binary cross-entropy. For
-`n_classes >= 3`, it uses the multi-class log loss function, with multinomial deviance
-and categorical cross-entropy as alternative names. The appropriate loss version is
-selected based on :term:`y` passed to :term:`fit`.
+Available losses for **regression** are:
+
+- 'squared_error', which is the default loss;
+- 'absolute_error', which is less sensitive to outliers than the squared error;
+- 'poisson', which is well suited to model counts and frequencies;
+- 'quantile', which allows for estimating a conditional quantile that can later
+  be used to predict confidence intervals.
```

Comment:
I would rather speak of "prediction intervals": the width of such intervals are not expected to shrink with a larger sample size (or training set size), because of the presence of aleatoric noise, that is the fact that Y can vary because of causes independent of what is measured in X.

On the contrary, the width of confidence intervals (of a given population quantity such as the population mean or a median, possibly conditioned on X) are expected to shrink with data sets sizes.

```suggestion
  be used to obtain prediction intervals.
```

A bit like standard deviation vs standard error of the mean.